### ISSUE TITLE
Design new Berlin banner

### ISSUE THREAD

**Issue by guylepage3** [blockstack/designs#612]
Design new Berlin banner

See.. https://github.com/blockstack/blockstack.org/issues/483

**Comment by guylepage3** [comment]
@pstan26 thoughts?
![v4 3 0-berlin-banner-01](https://user-images.githubusercontent.com/1711854/35051146-88fa8d9c-fb72-11e7-8c45-d08a2dacaceb.png)

**Comment by pstan26** [comment]
Woah nice, looks lovely. I think we are making the decision today to swap the name out to be "Blockstack Berlin: A Signature Fund Event". Shannon will send over the file with the new name.

**Comment by guylepage3** [comment]
@pstan26. So the logo will be changing?

**Comment by pstan26** [comment]
Correct, see this: https://www.eventbrite.com/e/blockstack-berlin-a-signature-fund-event-tickets-39425916979

**Comment by shanvoight** [comment]
@guylepage3 here are the new files for the event logo
![blockstackberlin_logos_blacknoborder](https://user-images.githubusercontent.com/29715942/35053722-ea753b8e-fb78-11e7-8014-aebf10d05ef8.png)
![blockstackberlin_logos_blackwithborder](https://user-images.githubusercontent.com/29715942/35053727-ec9851e4-fb78-11e7-8ecd-c3754a5ea173.png)
![blockstackberlin_logos_whitenoborder](https://user-images.githubusercontent.com/29715942/35053730-edcd271a-fb78-11e7-8a35-259d01bc8b42.png)
![blockstackberlin_logos_whitewithborder](https://user-images.githubusercontent.com/29715942/35053734-ef3a807a-fb78-11e7-9964-5f17be243792.png)
[BlockstackBerlin_Logos.pdf](https://github.com/blockstack/designs/files/1639725/BlockstackBerlin_Logos.pdf)

**Comment by shanvoight** [comment]
sent over email as well.

**Comment by guylepage3** [comment]
Thanks @shanvoight @pstan26 👍

**Comment by guylepage3** [comment]
@pstan26 here's the version with the updated logo. Let me know if we're good to go.

![v4 3 1-berlin-banner](https://user-images.githubusercontent.com/1711854/35075148-6bfa0e0a-fbbf-11e7-92aa-aa5e40ab5b5d.png)

**Comment by pstan26** [comment]
Looks good to me. Thank you

**Comment by pstan26** [comment]
cc'ing @shea256 too since he should sign off, as it is the homepage :-)

**Comment by shea256** [comment]
Seems a bit intrusive compared to the earlier token banner.

Question should be whether it's a net positive diverting from the rest of the core actions on the page and to what extent do you want to distract.

It'd be good to have an answer to that question before moving forward here.

**Comment by guylepage3** [comment]
>Seems a bit intrusive compared to the earlier token banner.

Well it's as big as the banner used in past Summit events. We can shrink it a bit for sure. It's currently at `110px` height on large screens, `80px` height on tablet and desktop, and smaller if possible on mobile.

As for attention on the page... We need weight in the value of the event vs the importance of the website content. I would say the proposed above height solutions should work. The banner currently is dark and non-invasive as it blends into the hero. Token banner was an electric blue that definitely caught the eye.

@shea256 let me know your thoughts and we can go from there.

**Comment by pstan26** [comment]
I'm biased but I think it may increase developer intrigue to build an app
on Blockstack, as we are establishing ourselves as the place where high
signal people are flocking to. What if we tried this out for a week where
when you click the link it opens a new tab where the event lives (as
opposed to navigating you to a different page​). if it doesn't drive ticket
sales and we also see users visiting tutorials page less frequently we can
alternate it back.

Also maybe we can make it a bit thinner and have a single color for a
background so its less intrusive?

**Comment by guylepage3** [comment]
@pstan26 @shea256 here's another version of the banner. Let me know your thoughts.

![v4 3 2-berlin-banner](https://user-images.githubusercontent.com/1711854/35154741-595d594c-fcf9-11e7-8336-aa1819f37830.png)

**Comment by shanvoight** [comment]
Looks great @guylepage3. This will help boost sales for the event and only be active for a few weeks. Would love to move forward with it today - if approved?

**Comment by guylepage3** [comment]
@shanvoight we're just waiting on approval from @shea256 @pstan26 and then @yknl will build it out.

**Comment by pstan26** [comment]
Ah there is another issue which is related to this: https://github.com/blockstack/blockstack.org/issues/483

**Comment by guylepage3** [comment]
<img width="1437" alt="screen shot 2018-01-23 at 6 27 34 pm" src="https://user-images.githubusercontent.com/1711854/35306368-415416a4-006b-11e8-8789-c5e2a8dda4b5.png">

**Comment by pstan26** [comment]
nice, understated and clean. love. thank you so much Guy
